daemon/algod/api/server/v2/handlers.go

```diff
@@ -883,6 +894,57 @@ func (v2 *Handlers) RawTransaction(ctx echo.Context) error {
 	return ctx.JSON(http.StatusOK, model.PostTransactionsResponse{TxId: txid.String()})
 }
 
+// SimulateTransaction simulates broadcasting a raw transaction to the network, returning relevant simulation results.
+// (POST /v2/transactions/simulate)
+func (v2 *Handlers) SimulateTransaction(ctx echo.Context) error {
+	if !v2.Node.Config().EnableExperimentalAPI {
+		// Right now this is a redundant/useless check at runtime, since experimental APIs are not registered when EnableExperimentalAPI=false.
+		// However, this endpoint won't always be experimental, so I've left this here as a reminder to have some other flag guarding its usage.
+		return ctx.String(http.StatusNotFound, fmt.Sprintf("%s was not enabled in the configuration file by setting EnableExperimentalAPI to true", ctx.Request().URL.Path))
+	}
+
+	stat, err := v2.Node.Status()
```

Comment:
@jasonpaulos For my understanding - We had a verbal discussing the possibility of hiding the API behind an opt-in feature flag.  Is the absence of a feature flag the intended state here?

* I'm mostly asking to confirm I'm follow along.  
* I realize `EnableExperimentalAPI` acts as an opt-in mechanism. However, it will eventually be removed leaving _no_ feature flag.

Reply:
Disregard - I didn't fully vet the comments several lines above.

OK - The inline comments seem reasonable enough.  Resolving.